packages/react-storage/src/components/StorageBrowser/views/hooks/usePaginate.ts

```diff
@@ -25,8 +26,11 @@ export const usePaginate = ({
     setCurrentPage(1);
   }).current;
 
-  return React.useMemo(
-    () => ({
+  return React.useMemo(() => {
+    const isInitialPage = currentPage === 1;
```

Comment:
Nit: Given that this is more or less "hardcoded" to mean page 1, would it be clearer to name this `isFirstPage`?